_posts/2019-02-26-foreman-community-newsletter---february-2019.md

```diff
@@ -0,0 +1,196 @@
+---
+layout: post
+title: Foreman Community Newsletter (February 2019)
+date: 2019-02-26 09:30:18
+author:
+  name: Greg Sutcliffe
+  uri: gwmngilfen
+tags:
+- foreman
+- newsletter
+- community
+excerpt: |
+  We're back! 2 months of goodies, including my departure, new releases,
+  conference roundup, and RFCs that need your input!
+---
+
+Welcome back to the Foreman Newsletter! We kinda slipped up in January, sorry
+about that, but we're back with 2 months of news, gossip, and releases for you!
+
+### Changes in the community team
+
+At the start of February, I made the difficult decision to step down as
+community manager. It's been an honour and a delight to help shape this
+community over the last few years, and to just be a part of it before that. You
+can read more about my (our?) history in my dedicated blog post [here][blog1]
+
+We can't, however, leave the community without a community lead, and so there
+has been some other changes to address this. I'm delighted to announce that Ori
+Rabin is [taking over as community lead][blog2] and Rahul Bajaj is joining the
+team, as well - read his introduction [here][blog3].
+
+I'm certain that along with Tomer, the new community team is ready to take
+Foreman on to bigger and better things!
+
+[blog1]: https://theforeman.org/2019/02/stepping-down.html
+[blog2]: https://theforeman.org/2019/02/introducing-the-new-community-team.html
+[blog3]: https://theforeman.org/2019/02/introducing-myself.html
+
+### Conference roundup
+
+As ever, Foreman was present at DevConf CZ, FOSDEM, and CfgMgmtCamp this year,
+and as usual had a hackday in Gent as well. This year was as busy as ever, with
+lots of traffic at the booths, lots of hats given out (wow, they were popular
+in the Brussels snowstorm...), and a great set of talks in our track at
```

Comment:
"we" also had talks in other tracks, is that smth worth mentioning?

Reply:
True indeed, thanks for all the fixes, will sort that.